mesonbuild/coredata.py

```diff
@@ -680,19 +773,21 @@ def parse_cmd_line_options(args):
     args.cmd_line_options = create_options_dict(args.projectoptions)
 
     # Merge builtin options set with --option into the dict.
-    for name, builtin in builtin_options.items():
-        names = [name]
-        if builtin.separate_cross:
-            names.append('cross_' + name)
-        for name in names:
-            value = getattr(args, name, None)
-            if value is not None:
-                if name in args.cmd_line_options:
-                    cmdline_name = BuiltinOption.argparse_name_to_arg(name)
-                    raise MesonException(
-                        'Got argument {0} as both -D{0} and {1}. Pick one.'.format(name, cmdline_name))
-                args.cmd_line_options[name] = value
-                delattr(args, name)
+    for name, builtin in chain(
```

Comment:
__[Flake8]__ [B007] Loop control variable 'builtin' not used within the loop body. If this is intended, start the name with an underscore. ([link](https://sider.review/gh/repos/19784232/pulls/5263#issue-4029502))

<sub>posted by Sider</sub>